nulib/images
# removing old fixtures, adding new ones

we're removing the old, weird fixtures and adding real records. We have 9 VRA Image records, 9 VRA Work records, and 3 Institutional Collections (each with 3 Multiresimages)

To test it out:
First make sure jetty isn't running on your laptop 
Rename your jetty directory in the Images project to jetty-old or something
Unzip the jetty-clean.zip that I gave you
start up jetty
run this rake task: rake hydra:fixtures:refresh

Then load up your rails server and look to see if the new records are showing up. If you log in, there should be 3 Institutional Collections, each with 3 images
